src/components/tab/Tab.tsx

```diff
@@ -1,11 +1,11 @@
-import { FC } from 'react'
+import React, { FC } from 'react'
```

Comment:
u already have destructure here, put SyntheticEvent there instead of importing react